### ISSUE TITLE
Aremi logo missing alt text

### ISSUE THREAD

**Issue by chloeleichen** [TerriaJS/terriajs#960]
The Aremi logos are missing alt text, which is a WCAG 2.0 AA requirement. 

![accessibility](https://cloud.githubusercontent.com/assets/7508939/10358882/c1abf226-6dde-11e5-8f6e-5532ac52250e.png)

**Comment by meh9** [comment]
@chloeleichen thanks for the bug, this should probably go over here in the AREMI repo: https://github.com/NICTA/aremi-natmap

:)

**Comment by chloeleichen** [comment]
Thanks @meh9 , closing this and moving to Aremi repo
